tests/models/test_model_names.py

```diff
@@ -70,6 +70,18 @@ def get_model_names(model_name_type: Any) -> Iterator[str]:
     openai_names = [f'openai:{n}' for n in get_model_names(OpenAIModelName)]
     bedrock_names = [f'bedrock:{n}' for n in get_model_names(BedrockModelName)]
     deepseek_names = ['deepseek:deepseek-chat', 'deepseek:deepseek-reasoner']
+    cloudflare_names = [
```

Comment:
As mentioned above, more modern names are missing. I'd rather NOT include this list here if we can't get it dynamically from the API or SDK.